aptos-move/aptos-gas/src/gas_meter.rs

```diff
@@ -528,3 +733,73 @@ impl AptosGasMeter {
         self.charge(cost).map_err(|e| e.finish(Location::Undefined))
     }
 }
+
+struct PrintVisitor;
+
+impl PrintVisitor {
+    fn indent(&self, depth: usize) {
+        print!(
+            "{}",
+            std::iter::repeat_with(|| ' ')
+                .take(depth * 4)
+                .collect::<String>()
+        );
+    }
+}
+
+impl move_vm_types::views::ValueVisitor for PrintVisitor {
+    #[inline]
```

Comment:
Why this inline all the time? I think we should leave this to the compiler.

Reply:
Hmm... this one should actually be removed since it's not used anywhere.

I think the other instances of `#[inline]` in aptos-gas are needed as we want to ensure those small routines are indeed getting inlined since they are called once every Move instruction.